pytorch_lightning/accelerators/ddp_accelerator.py

```diff
@@ -161,6 +161,26 @@ def _step(self, args):
             output = self.trainer.model(*args)
         return output
 
+    def backward(self, closure_loss, optimizer, opt_idx, *args, **kwargs):
+        automatic_optimization = self.trainer.train_loop.automatic_optimization
+
+        if not automatic_optimization:
+            # Manually prepare for reduce as user calling backwards manually
+            self.ddp_plugin.on_before_manual_backward(self.trainer.model, closure_loss)
+
+        super().backward(
+            closure_loss=closure_loss,
+            optimizer=optimizer,
+            opt_idx=opt_idx,
+            *args,
+            **kwargs
+        )
+
+        if not automatic_optimization:
+            # Manually prepare for reduce as user calling backwards manually
+            self.ddp_plugin.on_after_manual_backward(self.trainer.model)
+        return closure_loss
```

Comment:
all this looks very same, can we move it to the parent DDP accelertor?

Reply:
Done